### ISSUE TITLE
A "Timeline Options" tooltip appears under my pointer upon hover while recording.

### ISSUE THREAD

**Issue by paulpalinkas** [audacity/audacity#2869]
**Describe the bug**
A (possibly errant) "Timeline Options" tooltip appears under my pointer upon hover while recording.

**To Reproduce**

1. Commence recording using the 'R' hotkey.
2. Move mouse pointer anywhere, even outside the window of your current project. 
3. Wait .5s or so for hover condition.

![aud313-timelineoptions](https://user-images.githubusercontent.com/6475182/165813220-224216e4-9e9f-433d-b4be-1bd6147ab239.jpg)
**Actual behavior**
 "Timeline Options" help tag appears, will remain until you move and hover the mouse elsewhere. Clicking within the project window will dismiss the help tag.

**Expected behavior**
I’m unclear what the purpose of the help tag is, but in any case it's not contextual if I'm hovering over a window from another application.

**Additional information (please complete the following information):**
 - OS: macOS 12.3.1
 - Version: Audacity 3.1.3 
 
**Additional context**
This behavior was present in Audacity 2 as well. 

The reason this is an issue for me is because I am recording copy I’m reading from a browser window. I position my pointer over the window to scroll the page I am reading from, but I keep keyboard focus on the Audacity project window.  The help tag consequently appears over the very copy I am trying to record!

**Comment by paulpalinkas** [comment]
I just discovered I can reproduce this with a more general case.

1. Launch Audacity or open a new project window.
2. Move pointer anywhere without clicking, including outside project window. 
3. Wait .5 or so for hover state.

Clicking anywhere dismisses the help tag. 

I’ve also added some context for why this is bug is actually an issue for me.

**Comment by LWinterberg** [comment]
I cannot reproduce this on Windows/Linux - @dozzzzer can you repro this on your mac?

**Comment by dozzzzer** [comment]
Not right away but still I managed to reproduce the exact same behavior described by @paulpalinkas 
on MacOS 12.3.1 with Audacity version 3.1.3 

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/70063970/167155909-6e82b2c0-7ff3-4b41-a5c4-8d978ed4f474.png">

However, if you try to do the same with the latest 3.2.0 build, the observed behavior will be slightly different.
The tooltip does appear, though, its text depends on what the pointer has been moved over when moved off the project window. After I click the Record button and start moving the pointer to the left crossing Meter Toolbar, I see "Audacity Meter Toolbar" tooltip:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/70063970/167157348-c0529437-475f-4a26-887e-853473af3e62.png">

So overall, even though it's a bit different from what @paulpalinkas has reported, it's a very similar issue.
